### PR TITLE
Model Summary View Updates

### PR DESCRIPTION
**Pull request by krishnaglick**
## Change description

Model summary view in enterprise and community ui has better messaging and hidden buttons when no records are found.
Model summary view in config ui has import record functionality added.

### Development

- [X] Application changes have been tested appropriately

### Impact

- [x] Code follows company security practices and guidelines
- [x] Security impact of change has been considered
- [x] Performance impact of change has been considered
- [x] Possible migration needs considered (model migrations, config migrations, etc.)

Please explain any security, performance, migration, or other impacts if relevant:

### Code review

- [x] Pull request has a descriptive title and context useful to a reviewer. Screenshots or screencasts are attached where applicable.
- [x] Relevant tags have been added to the PR (bug, enhancement, internal, etc.)

### CODE REVIEW

**Comment by teallarson** [comment]
Tried running this locally to check it out  but got this on `/model/users/overview` after doing `roo demo -c` and `npm run dev` (community and enterprise both).  A few minutes later (when the sample record it was looking for was there maybe?) it rendered fine.  Running it with `roo config` the page rendered fine too!

Screen Shot 2022-01-12 at 2 57 46 PM

File: ui/ui-components/components/record/SampleRecordCard.tsx [hunk 1]
```diff
@@ -473,6 +493,9 @@ const SampleRecordCard: React.FC<SampleRecordCardProps> = ({
           A Sample Record can be used to validate your configuration is
           importing the correct data to Groups and Destinations.
         </p>
+        {!isConfigUI && (
+          <p>Run a schedule on your primary source to generate records.</p>
```
Comment (tealjulia) [inline ui/ui-components/components/record/SampleRecordCard.tsx:497]: Nitpick: I'd capitalize the Grouparoo object names:

```suggestion
          <p>Run a Schedule on your Primary Source to generate Records.</p>
```

File: ui/ui-components/__tests__/components/record/SampleRecordCard.tsx [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+  it("should render with no records", async () => {
+    cardProps.execApi = async () => ({ records: [], total: 0 } as any);
+    let card: ReturnType<typeof render>;
+    await act(async () => {
+      card = render(<SampleRecordCard {...cardProps} />);
+    });
+    expect(card.container).toMatchSnapshot();
+  });
+
+  it("should render with 1 record", async () => {
```
Comment (teallarson) [inline ui/ui-components/__tests__/components/record/SampleRecordCard.tsx:51]: Do we want to also test the new messaging (when there are mapped properties but no records have been imported yet)?

**Review by teallarson — APPROVED** [review]
Nice!  And it's great to be working towards front end test coverage.

Just one question on the testing.

File: ui/ui-components/__tests__/components/record/SampleRecordCard.tsx (hunk 2 above)
Comment (krishnaglick) [reply ui/ui-components/__tests__/components/record/SampleRecordCard.tsx:51]: Yeah, I should.

**Review by edmundito — APPROVED** [review]
Paired with @krishnaglick to review and LGTM

**Pull request merged**